client/src/components/layouts/Default.js

```diff
@@ -87,7 +93,13 @@ const mapStateToProps = createSelector(
 
 const mapDispatchToProps = dispatch =>
   bindActionCreators(
-    { fetchUser, removeFlashMessage, onlineStatusChange, executeGA },
+    {
```

Comment:
It seems that there was some automatic formatting involved during committing. I did not do this on purpose. Hope it's okay

Reply:
Yep, that's fine. Just let prettier do its thing and don't worry about formatting.